modules/ROOT/pages/index.adoc

```diff
@@ -1 +1,58 @@
-= Agent Visualizer
+= Agent Visualizer Overview
+
+Agent Visualizer provides a detailed visual graph so you can see, understand, and interact with your enterprise's evolving network of agents, Model Context Protocol (MCP) Servers, and LLMs. Agent Visualizer gives you insight into hidden agent interactions, governance gaps, and performance bottlenecks. Agent Visualizer offers complete situational awareness of your agentic ecosystem. 
+
+With Agent Visualizer, you can:
+
+* Get a clear view of connections and interactions among all your agents, clients, and Model Context Protocol (MCP) servers.
+* Understand data movement and dependencies at a glance by seeing which systems are calling your agents and what those agents are calling in turn.
+* Detect potential issues proactively.
+* Optimize agent orchestration for performance, trust, and scale. 
+
+Agent Visualizer is an intuitive user interface that helps you:
+
+ * Gain full visibility into your agent network by showing all connected agents in one network map, with clear relationships and direction of calls, gaining complete situational awareness of your agentic ecosystem. 
+ * See unclear interaction patterns by showing real-time or historical transaction volume overlays on network edges. This helps you understand key pathways and identify under or overused connections. 
+ * See governance gaps. You can see which agents are protected by Flex Gateways and policies. 
+ * Understand node purpose and origin. You gain full understanding of each asset in your agent network and any active running instances when you select a node. 
+ * Easily view product and non-production traffic. 
+
+[[use-cases]]
+== Use Cases
+
+Use Agent Visualizer to gain clarity and control over your agent ecosystem.
```

Comment:
For later - think about how to consolidate the bullet lists with maybe a more JTBD approach (like this one). Is there a way to move some of the other bullets as part of these?